src/pages/home/report/ReportActionCompose.js

```diff
@@ -236,6 +236,14 @@ class ReportActionCompose extends React.Component {
             this.focus(false);
         });
 
+        // This listener is used for focusing the composer again after going back to a report without remounting it.
+        // eslint-disable-next-line rulesdir/prefer-early-return
+        this.unsubscribeNavFocus = this.props.navigation.addListener('focus', () => {
+            if (this.willBlurTextInputOnTapOutside && !this.props.isFocused && !this.props.modal.isVisible) {
+                this.focus();
```

Comment:
Why disable the eslint check? Can we instead follow its instructions and return if the inverse of the condition is true?